GNUmakefile

```diff
@@ -87,7 +87,7 @@ dependency/sources/extract-libtermkey: dependency/sources/$(LIBTERMKEY).tar.gz
 	tar xzf $< -C $(dir $<)
 	touch $@
 
-dependency/sources/build-libtermkey: dependency/sources/extract-libtermkey
+dependency/sources/build-libtermkey: dependency/sources/extract-libtermkey dependency/sources/install-libncurses
```

Comment:
Because the build instructions make lua  link against ncurses...
Updating patches, stay tuned.